.travis.yml

```diff
@@ -31,8 +31,7 @@ before_script:
   - cd src/ontology
 
 script:
-  - echo "SKIPPING TRAVIS REAL BUILD TO FORCE SUCCESS"
-  #- ./run.sh make test
+  - ./run.sh make test
```

Comment:
I believe @ShahimEssaid deliberately skips the travis test. I will merge this PR and we can take out the test later if need be

Reply:
Yeap sorry,did that again.forgot that this was the whole reason we had the jenkins.